web/client/components/map/openlayers/DrawSupport.jsx

```diff
@@ -569,8 +569,19 @@ export default class DrawSupport extends React.Component {
                 this.props.onEndDrawing(feature, this.props.drawOwner);
                 feature = reprojectGeoJson(feature, this.getMapCrs(), "EPSG:4326");
 
-                const newFeatures = isSimpleGeomType(this.props.drawMethod) ?
-                    this.props.features.concat([{...feature, properties}]) :
+                const newFeatures = isSimpleGeomType(this.props.drawMethod) && this.props.features[0].geometry.type !== "GeometryCollection" ?
+                    this.props.features.map(feat => ({
```

Comment:
If you move this map after the concat, you don't have to rewrite featureProjection? 
by the way, why this is needed ? Probably it will have to be replaced with standard crs indication in geojson. Please take note of this in the issue you're going to open to refactor drawpsupport